src/button.test.jsx

```diff
@@ -1,35 +1,35 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import TestUtils from 'react-addons-test-utils';
-import Button from './Button';
-import Icon from './Icon';
+import { findComponentsWithType } from 'meetup-web-mocks/lib/testUtils';
 import { hasRoleAttribute, variantTest } from './utils/testUtils';
+import Button, { BUTTON_CLASS } from './Button';
+import Flex from './Flex';
+import Icon from './Icon';
 
 describe('Button', () => {
-	const BUTTON_CLASS = 'button';
-
 	describe('is a HTML button element', () => {
-		let buttonEl;
+		let button;
 
-		beforeEach(() => {
-			const button = TestUtils.renderIntoDocument(<Button />);
-			buttonEl = ReactDOM.findDOMNode(button);
+		beforeAll(() => {
```

Comment:
I very strongly recommend you leave this as `beforeEach` - there's no way to guarantee that _no_ future test will mutate (or even reassign) `button`, so `beforeAll` exposes the tests to a class of bugs that could lead to some significant unit test problems. Since the performance cost of `beforeEach` vs `beforeAll` is negligible, just use `beforeEach`